Package.swift

```diff
@@ -12,7 +12,7 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(name: "FinniversKit", url: "https://github.com/finn-no/FinniversKit.git", .branch("tjt-object-page"))
+        .package(name: "FinniversKit", url: "https://github.com/finn-no/FinniversKit.git", .upToNextMajor(from: "93.10.0"))
```

Comment:
Nice you remembered to use up to next major! 😉

Reply:
Got a good tip from a nice guy 😆